playbooks/roles/wireguard/templates/instructions-fr.md.j2

```diff
@@ -28,4 +28,4 @@ WireGuard n'est actuellement disponible que pour Linux, mais support [multi-plat
    `sudo wg-quick down wg0-client`
 
 #### Une note sur la configuration DNS ####
-Le fichier `wg0-client.conf` inlcure des commandes `PostUp` et `PostDown` simples qui utilisent `resolvconf` afin de diriger le trafic DNS vers le serveur dnsmasq qui est disponible via l'interface cryptée WireGuard à `{{ dnsmasq_wireguard_ip }}`. Bien que `resolvconf` soit un utilitaire commun, vous devrez peut-être modifier ces lignes pour votre distribution ou votre configuration réseau particulière.
+Le fichier `wg0-client.conf` inlcure le commande `DNS` qui utilise `resolvconf` afin de diriger le trafic DNS vers le serveur dnsmasq qui est disponible via l'interface cryptée WireGuard à `{{ dnsmasq_wireguard_ip }}`. Bien que `resolvconf` soit un utilitaire commun, vous devrez peut-être remplacer ce ligne avec `PostUp`/`PostDown` pour votre distribution ou votre configuration réseau particulière.
```

Comment:
Indeed I have no idea the gender of any nouns. Sometimes when I speak I just kind of blur the articles into a halfway sound and hope nobody notices my laziness.

Will fix and re-push.